LowVoltage/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM centos:centos7
+
+WORKDIR /usr/app
+
+RUN yum install -y epel-release
+RUN yum install -y python-pip
+RUN pip install --upgrade "pip < 21.0"
+
+#install the PyVISA-py PyVISA package via pip
+RUN pip install PyVISA-py PyVISA
+
+#install rpm & deltarpm  
+RUN yum install -y rpm
+RUN yum install -y deltarpm # otherwise: it complains : Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
+
+#install unzip  
+RUN yum install -y unzip 
+
+
+#add the rsvisa-centos_5.12.1-1.x86_64.rpm package (provided by Pierre)
+COPY rsvisa-centos_5.12.1-1.x86_64.rpm /usr/app/LowVoltage_packages/
```

Comment:
put all necessary files except the Dockerfile and the python script in a subdirectory, low_voltage/driver.
Then you can copy all of them in one go  

```
COPY driver . 
```

make sure to review and test the dockerfile accordingly